<launch>
	<arg name="framerate" default="30"  />
	<arg name="rgb_video" default="False"  />
	<arg name="rgb_video_file_path" default=""  />
	<arg name="rgb_preview" default="False"  />
	<arg name="rgb_preview_file_path" default=""  />
	<arg name="left" default="False"  />
	<arg name="left_file_path" default=""  />
	<arg name="right" default="False"  />
	<arg name="right_file_path" default=""  />
	<arg name="disparity" default="False"  />
	<arg name="disparity_file_path" default=""  />
	<arg name="depth" default="False"  />
	<arg name="convert_to_video" default="False"  />
	<arg name="qt_compatible" default="False"  />
	<node name="depthai_publish_save_streams" pkg="cv" exec="depthai_publish_save_streams" output="screen">
		<param name="framerate" value="$(var framerate)" />
		<param name="rgb_video" value="$(var rgb_video)" />
		<param name="rgb_video_file_path" value="$(var rgb_video_file_path)" />
		<param name="rgb_preview" value="$(var rgb_preview)" />
		<param name="rgb_preview_file_path" value="$(var rgb_preview_file_path)" />
		<param name="left" value="$(var left)" />
		<param name="left_file_path" value="$(var left_file_path)" />
		<param name="right" value="$(var right)" />
		<param name="right_file_path" value="$(var right_file_path)" />
		<param name="disparity" value="$(var disparity)" />
		<param name="disparity_file_path" value="$(var disparity_file_path)" />
		<param name="depth" value="$(var depth)" />
		<param name="convert_to_video" value="$(var convert_to_video)" />
		<param name="qt_compatible" value="$(var qt_compatible)" />
	</node>
</launch>
